firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -260,26 +266,29 @@ void state3_stillness(){
     Log.warn("door opened, session over, going from state3_stillness to idle");
     publishStateTransition(3, 0, checkDoor.doorStatus, checkINS.iAverage);
     saveStateChange(3, 2);
+    hasDurationAlertBeenSent = 0; // session is over, so reset flag
     stateHandler = state0_idle;
-
   }
-  else if(millis() - state2_duration_timer >= state2_max_duration){
+  else if((millis() - state2_duration_timer >= state2_max_duration)
+          && !hasDurationAlertBeenSent){
 
     Log.warn("See duration alert, going from state3 to idle after alert publish");
     publishStateTransition(3, 0, checkDoor.doorStatus, checkINS.iAverage);
```

Comment:
This is no longer accurate because you're doing to State 2 instead of idle

Reply:
I updated this in my latest commit